### ISSUE TITLE
`ex_doc` not recognized as command (Windows)

### ISSUE THREAD

**Issue by cw789** [starbelly/rebar3_ex_doc#1]
On Windows I get the following error:
```
===> Running ex_doc for gen_smtp
===> '.../gen_smtp/_build/default/plugins/rebar3_ex_doc/priv/ex_doc' is not recognized as an internal or external command,
operable program or batch file.
```
This could be fixed by having a `.../priv/ex_doc.cmd` with the following contents:

```
@echo off
escript.exe "%~dpn0" %*

```

That's how rebar3 recommends it for rebar3 itself: https://rebar3.readme.io/docs/getting-started#installing-binary

Sorry, can't send a PR right now.

**Comment by starbelly** [comment]
Thanks for opening this issue. We'll get this added.

**Comment by starbelly** [comment]
@cw789 I don't have access to a windows machine and testing this in CI at least on the surface is quite tricky as we have to work around uploading ex_doc from one job and downloading it another from a job that runs in windows, which sounds simple enough but github's upload artifact action errors out on the artifact being in a symlinked directory :) 

So, I think it might be easier and perhaps best if you can try something for me. 

If you would oblige : 

Go into your rebar3 project which has rebar3_ex_doc pulled in and start up a shell via rebar3 (`rebar3 shell`). 

Then run the following : 

```
rebar_utils:sh("escript.exe " ++ filename:nativename(filename:join(code:priv_dir(rebar3_ex_doc), "ex_doc")), [{return_on_error, true}])
```

That should get you something like `{error,{1, "Too few arguments....}}`. If you get that, then I will merge and release a patch for this.

**Comment by starbelly** [comment]
@cw789 Thanks :)

**Comment by starbelly** [comment]
Closed via #2 . v0.2.4 was released to hex which includes this patch.  ❤️

**Comment by cw789** [comment]
Perfect. Tested and works as expected. 💛
